packages/bot-web-ui/src/components/load-modal/__tests__/recent.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import userEvent from '@testing-library/user-event';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { isMobile } from '@deriv/shared';
```

Comment:
why are we still using isMobile @maryia-matskevich-deriv ?

Reply:
removed